config/routes.rb

```diff
@@ -1,37 +1,37 @@
-Rails
-  .application
-  .routes
-  .draw do
+Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 
   root "home#index"
 
-  resources :patients
-  resources :users
-  put "/users/:id/verify", to: "users#verify", as: :verify_user
-  # for assigning a users to a facility
-  put "/assign", to: "users#assign_facility", as: :assign_facility
-  # for removing a users from a facility
-  put "/unassign", to: "users#unassign_facility", as: :unassign_facility
-
   # get "/patients/:id/view/details/edit", to: "patients#family_details", as: :patient_details
   get "/dashboard", to: "dashboard#index", as: :dashboard
   get "/search", to: "search#index"
   get "/schedule", to: "schedule#index", as: :schedule
   post "/schedule", to: "schedule#schedule"
 
-  get "logout", to: "sessions#logout"
+  # patients
+  resources :patients
+  # disease summary
+  resources :patient_disease_summaries
+  get "/patients/:id/disease_history", to: "patient_disease_summaries#index", as: :disease_history
+  put "/patients/:id/patient_disease_summary/", to: "patient_disease_summaries#update"
+  # family_details
+  get "/patients/:id/family_details", to: "family_details#index", as: :family_details
+  get "patients/:id/family_details/all", to: "family_details#allMembers"
+  get "/patients/:patient_id/family_details/edit", to: "family_details#edit"
+  put "/patients/:id/family_details/", to: "family_details#update"
+  # patient information
+  get "/patients/:id/show/family_details", to: "patients#show", as: :show_family_details
+  get "patients/:id/show/personal_details", to: "patients#show", as: :show_personal_details
+  get "/patients/:id/show/disease_history", to: "patients#show", as: :show_disease_history
```

Comment:
can we use namespace to group these? 
Not sure tho.
something like:
```ruby
namespace :admin do  
    resources :posts, :comments
end
```



```

Reply:
@peeyush14goyal